schemas/context/profile-contact-crm.schema.json

```diff
@@ -0,0 +1,154 @@
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/profile-contact-crm",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Profile work details",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/profile"],
+  "description": "This mixin is used to capture information regarding person details in context of Business to Business contact.",
+  "definitions": {
+    "profile-contact-crm": {
+      "properties": {
+        "xdm:contactID": {
+          "title": "Contact Record Id",
+          "description": "Contact record id.",
+          "type": "string"
+        },
+        "xdm:sourceType": {
+          "title": "Contact Source Type",
+          "description": "Contact source type.",
+          "type": "string"
+        },		
+        "xdm:ownerID": {
+          "title": "Contact Owner id",
+          "description": "The ID of the owner of the account associated with this contact.",
+          "type": "string"
+        },
+        "xdm:accountID": {
+          "title": "Contact Account id",
+          "description": "ID of the account that’s the parent of this contact.",
+          "type": "string"
+        },
+        "xdm:userOwnerID": {
+          "title": "User Owner Id",
+          "description": "User owner id",
+          "type": "string"
+        },
+        "xdm:deleted": {
+          "title": "Deleted Indicator",
+          "description": "Indicates whether the contact has been deleted or not .",
+          "type": "boolean"
+        },
+        "xdm:emailBouncedDate": {
+          "title": "Email Bounced Date",
+          "description": "If bounce management is activated and an email is sent to a contact, provides the date when the email was bounced.",
+          "type": "string",
+          "format": "date-time"
+        },
+        "xdm:emailBouncedReason": {
+          "title": "Email Bounced Reason",
+          "description": "If bounce management is activated and an email is sent to a contact, provides the reason for bounced email.",
+          "type": "string"
+        },
+        "xdm:emailBounced": {
+          "title": "Email Bounced Indicator",
+          "description": "If bounce management is activated and an email is sent to a contact, indicates whether the email bounced (true) or not (false).",
+          "type": "boolean"
+        },
+        "xdm:leadSource": {
+          "title": "Lead Source",
+          "description": "Field recording what source the lead originated from.",
+          "type": "string"
+        },
+        "xdm:contactDescription": {
+          "title": "Contact Description",
+          "description": "Contact description.",
+          "type": "string"
+        },
+        "xdm:contactStatus": {
+          "title": "Contact Status",
+          "description": "Current status of the contact.",
+          "type": "string"
+        },
+        "xdm:contactStatusDetail": {
```

Comment:
Check if this property has what significance.

Reply:
This is the "status_detail" property from the mkt_person entity.

Reply:
remove